### PR TITLE
Filter categories

### PR DESCRIPTION
**Pull request by madeuz**
closes CRJI/EIC#714

### CODE REVIEW

**Comment by gabriel-v** [comment]
Looking wonderful!

I didn't get too much time to review this, but here are some first remarks:
- Center the categories vertically (so the top category "COLLECTIONS" doesn't fall under the shade of the blue bar)
- A fix to the scroll-in-scroll issue: let's fix it by only displaying a single expanded list
  - E.g. Intella only displays one list at a time, and you select which list as [a drop-down](https://github.com/CRJI/EIC/blob/master/intella-screenshots/2021-03-22/Screenshot%202021-03-22%20at%2014.44.14.png)
  - Instead of some drop-down, let's accordion the bucket lists so that only one full list is opened; 
  - the lists that come before the open one go at the top (all closed)
  - and the ones that go below it are pinned at the bottom (all closed)
  - some list is always opened when you look at the category. When first opening the category, pick the first one that has filters set, and if no filters are set, open the very first one. 
  - When clicking around the categories, remember which list was expanded. This should mean accidental clicks are less annoying.

**Comment by gabriel-v** [comment]
For some other future work:



- Center the categories vertically (so the top category "COLLECTIONS" doesn't fall under the shade of the blue bar)
- A fix to the scroll-in-scroll issue: let's fix it by only displaying a single expanded list
  - E.g. Intella only displays one list at a time, and you select which list as [a drop-down](https://github.com/CRJI/EIC/blob/master/intella-screenshots/2021-03-22/Screenshot%202021-03-22%20at%2014.44.14.png)
  - Instead of some drop-down, let's accordion the bucket lists so that only one full list is opened; 
  - the lists that come before the open one go at the top (all closed)
  - and the ones that go below it are pinned at the bottom (all closed)
  - some list is always opened when you look at the category. When first opening the category, pick the first one that has filters set, and if no filters are set, open the very first one. 
  - When clicking around the categories, remember which list was expanded. This should mean accidental clicks are less annoying.

**Comment by madeuz** [comment]
two exceptions: 
- the ones that go below it are pinned at the bottom (all closed)
**_happens only if scroll appears, if previous list fits it just go below_**
- when there's >1 filter, do nothing for now **OR** if no filters are set, open the very first one
**_for now if there's > 1 filter and nothing is selected all filters stay closed_**

File: src/constants/aggregationFields.js [hunk 1]
```diff
@@ -141,16 +175,23 @@ export const aggregationFields = {
...
         filterLabel: 'Has Attachments',
         chipLabel: 'Has Attachments',
         type: 'term-or',
     },
     pgp: {
+        category: 'email',
+        categoryLabel: 'Email',
         filterLabel: 'PGP Encrypted',
         chipLabel: 'PGP Encrypted',
         type: 'term-or',
```
Comment (gabriel-v) [inline src/constants/aggregationFields.js:197]: reminder -- move the duplicated `categoryLabel` and `categoryIcon` to a different constant list in `src/constants`

**Review by gabriel-v — APPROVED** [review]

**Pull request merged**